salt/matchers/confirm_top.py

```diff
@@ -10,6 +10,14 @@
 log = logging.getLogger(__file__)
 
 
+def _load_matchers(opts):
+    """
+    Store matchers in __context__ so they're only loaded once
+    """
+    __context__["matchers"] = {}
```

Comment:
Can we remove `__context__["matchers"] = {}`

Reply:
Done. Thanks!